_posts/elements/2015-05-12-forms.md

```diff
@@ -0,0 +1,527 @@
+                    <option value="Violet">Violet</option>
+                </select>
+            </div>
+
+            <div class="field">
+                <label class="field-label" for="select_02">Choose your favorite color of the rainbow (enhanced select menu)</label>
+                <select class="field-input input-select replace-select" id="select_02" name="select_02">
+                    <option value="">- Choose one -</option>
+                    <option value="Red">Red</option>
+                    <option value="Orange">Orange</option>
+                    <option value="Yellow">Yellow</option>
+                    <option value="Green">Green</option>
+                    <option value="Blue">Blue</option>
+                    <option value="Indigo">Indigo</option>
+                    <option value="Violet">Violet</option>
+                </select>
+            </div>
+            <div class="field">
+                <label class="field-label" for="select_02_err">Choose your favorite color of the rainbow (enhanced select menu with error)</label>
+                <select class="field-input input-select replace-select has-error" id="select_02_err" name="select_02_err">
+                    <option value="">- Choose one -</option>
+                    <option value="Red">Red</option>
+                    <option value="Orange">Orange</option>
+                    <option value="Yellow">Yellow</option>
+                    <option value="Green">Green</option>
+                    <option value="Blue">Blue</option>
+                    <option value="Indigo">Indigo</option>
+                    <option value="Violet">Violet</option>
+                </select>
+                <div class="field-message has-error">
+                    <span class="field-message-content">
+                        You must select your favorite color.
+                    </span>
+                </div>
+            </div>
+            <div class="field">
+                <label class="field-label" for="select_02_suc">Choose your favorite color of the rainbow (enhanced select menu with valid selection)</label>
+                <select class="field-input input-select replace-select has-success" id="select_02_suc" name="select_02_suc">
+                    <option value="">- Choose one -</option>
+                    <option value="Red">Red</option>
+                    <option value="Orange">Orange</option>
+                    <option value="Yellow">Yellow</option>
+                    <option value="Green">Green</option>
+                    <option value="Blue" selected="selected">Blue</option>
+                    <option value="Indigo">Indigo</option>
+                    <option value="Violet">Violet</option>
+                </select>
+            </div>
+            <div class="field">
+                <label class="field-label" for="select_02_suc">Choose your favorite color of the rainbow (enhanced select and disabled menu)</label>
```

Comment:
The for value is the same as the above select "select_02_suc" - not sure if that matters in this example?